src/components/ArtistsSearchForm/ArtistsSearchForm.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import ArtistsSearchForm from './ArtistsSearchForm';
+
+describe('ArtistsSearchForm component', () => {
+  afterEach(() => cleanup());
+  it('renders ArtistsSearchForm', () => {
+    const { asFragment } = render(<ArtistsSearchForm 
+      handleSearch={() => {}}
```

Comment:
alternative syntax. jest.fn()
https://medium.com/@deanslamajr/jest-fn-all-the-things-d26f3b929986